saleor/checkout/calculations.py

```diff
@@ -347,6 +347,25 @@ def update_prior_unit_price_for_lines(lines: Iterable["CheckoutLineInfo"]):
             line_info.line.prior_unit_price_amount = line_info.prior_unit_price_amount
 
 
+def _is_checkout_modified(
```

Comment:
It is only suggestion, but I would suggest to add to the name something like `async` or `second_worker`. Checkout can be also modified by this worker, so looking only by name it is really hard to understand what is expected result.

Reply:
It's not `async` so should not be named as such.